Apps/Database/src/Assets/docs/EmailValidationTemplate.html

```diff
@@ -22,6 +22,7 @@ <h1 style = "font-size:18px;">Almost there!</h1>
                 <p>We've received a request to register your email address for a Health Gateway account.</p>
                 <p>To activate your account, please verify your email by clicking the link:</p>
                 <a style = "color:#1292c5;font-weight:600;" href = "${ActivationHost}/ValidateEmail/${InviteKey}"> Health Gateway Account Verification </a>
+                <p>This email verification link will expire in ${ExpiryHours} hours.</p>
```

Comment:
I don't agree with this naming. Ideally the same name should be used across our code, db and configuration. Using `ExpiryHours` here may be ok as we know the context , but as we need to use this name also in the WebConfig, I suggest to give it a more meaningful name such as EmailVerificationExpiryHours as `ExpiryHours` in the WebConfig doesn't say much about this context.

Reply:
The config and the variable don't have to have the same name, in this case the config is stored in seconds and the email displays it in hours, this means they will not have the same name anyway.

I'm following the standard for the other variables, eg. its InviteKey and not EmailVerificationInviteKey

Reply:
resolved per our team discussion, approved